docs/understand/enterprise/orchestration.md

```diff
@@ -0,0 +1,389 @@
+1. A proxy server should be configured to route traffic between the 2 servers.
+
+
+There are several ways to run lakeFS Enterprise. You can follow the examples below,
+or create your own setup using our resources available in our documentation.
+
+## Using lakeFS Helm chart
+
+With every new release of lakeFS or Fluffy, the lakeFS team releases a new [lakeFS 
+helm chart](https://artifacthub.io/packages/helm/lakefs/lakefs). Together with the granted
+Fluffy token, you can run the full lakeFS Enterprise solution.
+
+As an example, the following `values` file will run lakeFS Enterprise with OIDC integration.
+
+```yaml
+lakefsConfig: | 
+  logging:
+      level: "INFO"
+  blockstore:
+    type: s3
+  database:
+    type: postgres
+    postgres:
+      connection_string: <postgres-connection-string>
+  auth:
+    oidc:
+      # the claim that's provided by the OIDC provider (e.g Okta) that will be used as the username according to OIDC provider claims provided after successful authentication
+      friendly_name_claim_name: "<some-oidc-provider-claim-name>"
+      default_initial_groups: ["Developers"]
+    ui_config:
+      login_cookie_names:
+        - internal_auth_session
+        - oidc_auth_session
+ingress:
+  enabled: true
+  ingressClassName: <class-name>
+  hosts:
+    # the ingress that will be created for lakeFS
+    - host: <lakefs.ingress.domain>
+      paths: 
+       - /
+
+##################################################
+########### lakeFS enterprise - FLUFFY ###########
+##################################################
+
+fluffy:
+  enabled: true
+  image:
+    repository: treeverse/fluffy
+    tag: '0.4.0'
+    pullPolicy: IfNotPresent
+    privateRegistry:
+      enabled: true
+      secretToken: <dockerhub-token-fluffy-image>
+  fluffyConfig: |
+    logging:
+      format: "json"
+      level: "INFO"
+    database:
+      type: postgres
+      postgres:
+        connection_string: <postgres-connection-string>
+    auth:
+      serve_listen_address: 0.0.0.0:9000
+      logout_redirect_url: https://oidc-provider-url.com/logout/example
+      oidc:
+        enabled: true
+        url: https://oidc-provider-url.com/
+        client_id: <oidc-client-id>
+        callback_base_url: https://<lakefs.ingress.domain>
+        is_default_login: true
+        # the claim name that represents the client identifier in the OIDC provider (e.g Okta)
+        logout_client_id_query_parameter: client_id
+        # the query parameters that will be used to redirect the user to the OIDC provider (e.g Okta) after logout
+        logout_endpoint_query_parameters:
+          - returnTo
+          - https://<lakefs.ingress.domain>/oidc/login
+  secrets:
+    create: true
+  sso:
+    enabled: true
+    oidc:
+      enabled: true
+      # secret given by the OIDC provider (e.g auth0, Okta, etc)
+      client_secret: <oidc-client-secret>
+  rbac:
+    enabled: true
+
+useDevPostgres: true
+```
```

Comment:
It'll be hard to keep that "values example" in-sync. 
I think that only one source of truth of examples for Helm should exist.
Either here or in Github. IMO keep the [github](https://github.com/treeverse/charts/tree/master/examples/lakefs/enterprise) one in sync since it's close to the chart and related to it.

Reply:
Synced offline, decided to keep as is